### DISCUSSION TITLE
[Question][Hw4]  Gradescope reports coverage is below threshold

### DISCUSSION THREAD

**Discussion by HoseongLee** [prosyslab-classroom/cs524-program-analysis#299] [Q&A]
Name: Hoseong Lee

After implementing Hw4 and submitting to gradescope, I am getting an error "Coverage Below Threshold!!".
However, in the server provided, running make coverage results in a coverage of 71%. Which is higher than the required 70%
listed in the README. Is the coverage calculation different in the gradescope or am I missing something?
Any help would be much appreciated

**Comment by goodtaeeun** [comment]
I am sorry for the confusion. The threshold is 80%. I will make an announcement.
